### PR TITLE
Add table on meeting join and chat behavior

### PR DESCRIPTION
**Pull request by MicrosoftGordon**
Reviewed with Serdar Soysal to add 2 tables to further explain meeting join behavior and meeting chat behavior for Education.
Content was reviewed and approved in August, "Hi Gordon, this looks great. Please go ahead and update in GitHub. Do let me know if you run into issues. Markdown tables can get a little finicky. "
Content: https://microsoft.sharepoint.com/:w:/t/skypespacesteamnew/EaS_QgsxtaNLrOJpQd9EX6cB88ALFP8bX6OstNp7PGv5KA?e=lA3lyN

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by SerdarSoysal** [comment]
@scanum I reviewed this PR and signed off on it already. This is ready to merge post copy edit.

File: Teams/policy-packages-edu.md [hunk 1]
```diff
@@ -167,10 +167,31 @@ Turning off **Allow Meet now in channels**, **Allow channel meeting scheduling**
...
+
+| Meeting creation policy | Create a meeting | Start a meeting unattended | Bypass lobby when joining | End the meeting |
+| --- | --- | --- | --- | --- |
+| **On (e.g. educator)** | Yes | Yes | Determined by [meeting options](https://go.microsoft.com/fwlink/?linkid=2093366) | Yes, as organizer
+| **Off (e.g. student)** | No | No\*\* | Determined by [meeting options](https://go.microsoft.com/fwlink/?linkid=2093366) | No
+
+> [!NOTE]
+> \*\* This applies to EDU licensed users only and applies to meetings, channel meetings, instant meetings, and instant channel meetings.
+
 When you change the **Allow chat in meetings** policy to disabled and block students from scheduling meetings from above while and keep this policy on for educators (for the meetings that are not scheduled from a channel or meet now in a channel), students won't be able to chat before the educator joins the meeting, nor after the meeting. They will still be able to see the chat history before, during, and after the meeting. As an example, they'll be able to see messages from the teacher, or the meeting recording link, if the meeting was recorded.
```
Comment (JohanFreelancer9) [inline Teams/policy-packages-edu.md:180]:
```suggestion
When you change the **Allow chat in meetings** policy to disabled and block students from scheduling meetings from above while keeping this policy on for educators (for the meetings that are not scheduled from a channel or meet now in a channel), students won't be able to chat before the educator joins the meeting, nor after the meeting. They will still be able to see the chat history before, during, and after the meeting. As an example, they'll be able to see messages from the teacher, or the meeting recording link, if the meeting was recorded.
```

File: Teams/policy-packages-edu.md [hunk 2]
```diff
@@ -167,10 +167,31 @@ Turning off **Allow Meet now in channels**, **Allow channel meeting scheduling**
...
 
 If both students and educators have the **Allow chat in meetings** policy turned off, no one will be able to chat in the meeting chat window. The safety measure on meeting chat restriction described above only applies to education customers in Teams based on users’ license type.
 
+Here is a table to describe the logic for allow chat in meetings:
```
Comment (JohanFreelancer9) [inline Teams/policy-packages-edu.md:184]:
```suggestion
Here is a table to describe the logic for allowing chat in meetings:
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**